build.gradle

```diff
@@ -183,24 +183,10 @@ repositories {
     maven { url "https://aws.oss.sonatype.org/content/repositories/snapshots" }
 }
 
-configurations {
```

Comment:
Can you also pull in the latest changes on the main branch? This looks likes it from a former PR and not from the changes introduced in this PR.